### ISSUE TITLE
NSLayoutConstraint warning when running mpv on macOS Big Sur terminal

### ISSUE THREAD

**Issue by filcode** [mpv-player/mpv#8674]
### Important Information

- mpv version: 0.33
- macOS Version: macOS Big Sur 11.2.3
- Source of the mpv binary or bundle: https://laboratory.stolendata.net/~djinn/mpv_osx/

### Reproduction steps

`cd /Applications/mpv.app/Contents/MacOS `
`./mpv "https://www.youtube.com/watch?v=xVWs0ti0J90"`

### Warning that I get

`2021-03-24 01:41:21.747 mpv[1108:34617] This NSLayoutConstraint is being configured with a constant that exceeds internal limits.  A smaller value will be substituted, but this problem should be fixed. Break on BOOL _NSLayoutConstraintNumberExceedsLimit(void) to debug.  This will be logged only once.  This may break in the future.
2021-03-24 01:41:21.755 mpv[1108:34617] Warning: Expected min height of view: (<NSButton: 0x7fea7004f750>) to be less than or equal to 30 but got a height of 32.000000. This error will be logged once per view in violation.
2021-03-24 01:41:21.756 mpv[1108:34617] Warning: Expected min height of view: (<NSButton: 0x7fea6ed65d10>) to be less than or equal to 30 but got a height of 32.000000. This error will be logged once per view in violation.
2021-03-24 01:41:21.757 mpv[1108:34617] Warning: Expected min height of view: (<NS
[mpv.conf.zip](https://github.com/mpv-player/mpv/files/6193790/mpv.conf.zip)
Button: 0x7fea6ed68c20>) to be less than or equal to 30 but got a height of 32.000000. This error will be logged once per view in violation.`

### Screenshot
<img width="842" alt="Screen Shot 2021-03-24 at 01 41 43" src="https://user-images.githubusercontent.com/81129688/112241558-5fec8b80-8c42-11eb-9cbb-3d6ac2ec4c6d.png">

### mpv.conf file
[mpv.conf.zip](https://github.com/mpv-player/mpv/files/6193794/mpv.conf.zip)

**Comment by Akemi** [comment]
try master. i fixed something regarding this here that is not on 0.33 https://github.com/mpv-player/mpv/commit/aa00ad06aae264796a84327f1716534b4f37c622#diff-6bca6a2ac4c14d01b3c108041a8910a84193a72a1b4df28fcee00311a5345580

if those are new warning, fuck apple, and i will reopen it.

**Comment by johsoderi** [comment]
I get this warning on 0.33.1 when running from terminal:
`2021-11-12 14:27:49.410 mpv[99771:67281077] This NSLayoutConstraint is being configured with a constant that exceeds internal limits.  A smaller value will be substituted, but this problem should be fixed. Break on BOOL _NSLayoutConstraintNumberExceedsLimit(void) to debug.  This will be logged only once.  This may break in the future.`

OS version: macOS Big Sur 11.6.1
MPV version: 0.33.1 (installed via homebrew)

I'm not sure whether it's related or if I should create a new issue?

**Comment by Akemi** [comment]
like i said, it's not in 0.33 and 0.33.1 is just a bug fix version of 0.33 with one specific commit added.
https://github.com/mpv-player/mpv/compare/v0.33.0...v0.33.1
